docker/openproblems-r-extras/Dockerfile

```diff
@@ -1,19 +1,32 @@
 FROM singlecellopenproblems/openproblems-r-base:latest
 
+
 ARG NB_USER="sagemaker-user"
 ARG NB_UID="1000"
 ARG NB_GID="100"
 
 USER root
 WORKDIR /
 
+RUN apt-get update && apt-get install -y libhdf5-dev hdf5-tools \
+&& apt-get clean autoclean && apt-get autoremove --yes && rm -rf /var/lib/{apt,dpkg,cache,log}/
+RUN apt-get install -y libhdf5-dev=1.10.6+repack-4+deb11u1
+
 # Install R packages
 COPY ./docker/openproblems-r-extras/r_requirements.txt ./r_requirements.txt
-RUN sudo R -e "renv::install(scan('r_requirements.txt', sep='\n', what=character()))"
+RUN \
+    R -e "install.packages('Matrix')" \
+    && R -e "renv::install(scan('r_requirements.txt', sep='\n', what=character()))" \
+    && R -e 'install.packages("devtools")' \
```

Comment:
why not add `Matrix` and `devtools` to the `r_requirements.txt` file?

Reply:
For `Matrix` the order of installation didn't work correctly anymore and I got an error for some other package.

Reply:
I originally had `devtools` in a separate RUN instruction as I didn't want to rebuild all r packages. I later merged it into one run instruction so it would probably make sense to include that in the requirements file

Reply:
for `Matrix` this makes sense, as it's a dependency for other packages I guess. Agreed for devtools. Would be great to clean this up :)

Reply:
Does it change the behaviour if we put `Matrix` at the top of the file? I'd like to only circumvent `r_requirements.txt` if we absolutely must.